subprojects/language-java/src/main/java/org/gradle/api/tasks/javadoc/InternalJavadocDocletOptions.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks.javadoc;
+
+import org.gradle.external.javadoc.StandardJavadocDocletOptions;
+import org.gradle.external.javadoc.internal.JavadocOptionFile;
+
+class InternalJavadocDocletOptions extends StandardJavadocDocletOptions {
+    public InternalJavadocDocletOptions(StandardJavadocDocletOptions options) {
+        super(options);
+    }
+
+    public JavadocOptionFile getJavadocOptionFile() {
```

Comment:
Should this also include `@Deprecated` and have javadoc stating not to use it?

Reply:
See class level answer